Sources/IRGen/Expression/IRExpression.swift

```diff
@@ -8,6 +8,20 @@
 import AST
 import Lexer
 
+struct GeneratedCode {
+  var preamble: String
+  var expression: String
+
+  init(_ preamble: String, _ expression: String) {
```

Comment:
Is there any particular reason we don't want to explicitly label these arguments? I think it would be better to do so, especially with an initializer that takes two `String` parameters. Otherwise should we ever add or move these around we might inject preambles into bodies and vice versa.